docs/apache-airflow/concepts/dags.rst

```diff
@@ -140,28 +140,41 @@ You can also provide an ``.airflowignore`` file inside your ``DAG_FOLDER``, or a
++==========================================+======================================================================================================================+======================================+
+| ``None``                                 | No schedule, use for manually triggered DAGs                                                                         | ``None``                             |
++------------------------------------------+----------------------------------------------------------------------------------------------------------------------+--------------------------------------+
+| Cron expression (``str``)                | To run at cron-based intervals                                                                                       + ``"0 0 * * *"``                      |
++------------------------------------------+----------------------------------------------------------------------------------------------------------------------+--------------------------------------+
+| Cron preset (``str``)                    | Convenience cron expression for readability                                                                          + ``"@daily"``                         |
++------------------------------------------+----------------------------------------------------------------------------------------------------------------------+--------------------------------------+
+| List of cron expressions/presets         | To run at intervals that cannot be expressed by a single cron expression.                                            + ``["0 3 * * *", "0 0 * * MON,TUE"]`` |
++------------------------------------------+----------------------------------------------------------------------------------------------------------------------+--------------------------------------+
+| ``datetime.timedelta``                   | To run at frequency-based intervals. Useful if your interval cannot be expressed by cron e.g. ``timedelta(days=3)``. + ``timedelta(days=3)``                |
++------------------------------------------+----------------------------------------------------------------------------------------------------------------------+--------------------------------------+
+| ``dateutil.relativedelta.relativedelta`` | To express an interval in weeks, months, or years (which timedelta cannot do natively).                              + ``relativedelta(months=1)``          |
++------------------------------------------+----------------------------------------------------------------------------------------------------------------------+--------------------------------------+
+
+.. tip::
+    A convenient tool for converting cron expressions to human language is `Crontab.guru <https://crontab.guru/>`_.
 
-    If ``schedule_interval`` is not enough to express the DAG's schedule, see :doc:`Timetables </howto/timetable>`.
-    For more information on ``logical date``, see :ref:`data-interval` and
```

Comment:
This link seems worth keeping

Reply:
`logical_date` is not mentioned anywhere above, nor is there any text leading up to anything about `logical_date`. I don't see where it fits in here.

Reading again, we could move it down, between the last sentence of this section and the header "DAG Assignment".

Reply:
Moved it down, after `logical_date` was explained.